frontend/lib/norent/info.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Page from "../ui/page";
+import { OutboundLink } from "../analytics/google-analytics";
+import { LandingPagePartnerLogos, getImageSrc } from "./homepage";
+import { StaticImage } from "../ui/static-image";
+
+export const NorentInfoPage: React.FC<{}> = () => (
+  <Page title="NoRent | About Your Letter" className="content">
```

Comment:
Ohhh you are totally bringing up a BUG in our current code!!!  I gotta file it.  Right now the `<Page>` element assumes the `JustFix.nyc` site and automatically puts `JustFix.nyc` at the beginning of all pages, I think, but we should change it to use the name of the currently active site.  Also you shouldn't have to put `NoRent |` at the beginning of all these page titles, the `<Page>` element should basically do that for you.

Reply:
Nice, good to know